<?xml version="1.0" ?>
<launch>
    <node name="localization" pkg="robot_localization" type="ukf_localization_node">
        <param name="world_frame" value="map"/>
        <param name="map_frame" value="map"/>
        <param name="odom_frame" value="odom"/>
        <param name="baselink_frame" value="fcu_local"/>
        <param name="publish_tf" value="true"/>
        <param name="imu0" value="/mavros/imu/data"/>
        <rosparam param="imu0_config">[false, false, false, true, true, true, false, false, false, true, true, true, true, true, true]</rosparam>
        <param name="odom0" value="/mavros/global_position/local"/>
        <rosparam param="odom0_config">[true, true, true, false, false, false, false, false, false, false, false, false, false, false, false]</rosparam>
        <param name="gps0" value="/mavros/global_position/global"/>
        <rosparam param="gps0_config">[false, false, false, true, true, true, false, false, false, true, true, true, true, true, true]</rosparam>
    </node>
    <node name="navstat_transform" pkg="robot_localization" type="navsat_transform_node">
        <param name="broadcast_utm_transform" value="true"/>
    </node>
</launch>
